CMakeLists.txt

```diff
@@ -135,7 +135,9 @@ find_package (Eigen3 3.3 REQUIRED NO_MODULE)
 find_package (XLNT REQUIRED)
 
 ### Libigl set include path
-include_directories("${CMAKE_PREFIX_PATH}/../build/libigl/include/")
+find_package(LIBIGL REQUIRED QUIET)
```

Comment:
This does not configure, I get the following error

```
CMake Error at CMake/FindLIBIGL.cmake:34 (include):
  include could not find load file:

    libigl
Call Stack (most recent call first):
  CMakeLists.txt:138 (find_package)
```
I think the FindLIBIGL.cmake file is not written in a way that it can find the cloned libigl from a custom location.
What do you think @akenmorris @cchriste 

Thanks

Reply:
Ok it works if I specify the include directory in the cmake, as follows: 
`/home/sci/riddhishb/Documents/git-repos/ShapeWorksUser/ShapeWorks/dependencies/install/libigl/cmake/`

Reply:
Do you mean something like `cmake ... -DLIBIGL_DIR=/path/to/install/libigl/cmake ...`?